### PR TITLE
Improve libFuzzer error messages

### PR DESCRIPTION
**Pull request by inferno-chromium**
- Add fully qualified fuzzer name for easy debugging.
- Seperate out common dictionary failure error message.

### CODE REVIEW

File: src/python/bot/fuzzers/libFuzzer/engine.py [hunk 1]
```diff
@@ -226,12 +229,25 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         artifact_prefix=reproducers_dir,
         extra_env=options.extra_env)
 
-    if (not environment.get_value('USE_MINIJAIL') and
-        fuzz_result.return_code == constants.LIBFUZZER_ERROR_EXITCODE):
+    project_qualified_fuzzer_name = (
```
Comment (oliverchang) [inline src/python/bot/fuzzers/libFuzzer/engine.py]: nit: can you put this in its own method? it's starting to look a little cluttered here :)

File: src/python/bot/fuzzers/libFuzzer/engine.py [hunk 2]
```diff
@@ -226,12 +229,25 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
...
+        data_types.fuzz_target_project_qualified_name(
+            utils.current_project(), os.path.basename(target_path)))
+    dict_error_match = DICT_PARSING_FAILED_REGEX.match(fuzz_result.output)
```
Comment (oliverchang) [inline src/python/bot/fuzzers/libFuzzer/engine.py]: will this always be the very first thing in the output? i.e. if this occurs in the second line we won't catch it.

Comment (inferno-chromium) [reply src/python/bot/fuzzers/libFuzzer/engine.py]: Good catch, changed to re.search.

File: src/python/bot/fuzzers/libFuzzer/engine.py (hunk 1 above)
Comment (inferno-chromium) [reply src/python/bot/fuzzers/libFuzzer/engine.py]: Done!

**Pull request merged**